package.json

```diff
@@ -104,23 +174,88 @@
 				}
 			],
 			"commandPalette": [
+				{
```

Comment:
The activity bar in the navigation panel still seems to be there (wrong line I know).